задание 6 1 вариант 9/Program.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace задание_6_1_вариант_9
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+                Console.WriteLine("Введите число,кратное 4 и не оканчивающееся нулем");
```

Comment:
Можно вводить любое целое число!
```suggestion
                Console.WriteLine("Введите целое число");
```